setup.cfg

```diff
@@ -41,10 +41,11 @@ project_urls =
 packages = find:
 install_requires =
     appdirs>=1.4.3,<2
+    backports.entry_points_selectable
```

Comment:
You must lower pin dependencies 🤔 Why do we need this and can't use importlib-metadata?

Reply:
See https://pypi.org/project/backports.entry-points-selectable/ for description of why it's needed.

Reply:
For Virtualenv, basically it boils down to supporting older Pythons (3.5, 2.7). I'll add a comment.